templates/includes/related-content.html

```diff
@@ -12,8 +12,8 @@ <h3 class="sr-only">Event - ‘Before Shakespeare’ at The National Archives</h
                 </picture>
             </div>
             <div class="card-group-secondary-nav__body">
-                <h3 class="tna-card__heading" aria-hidden="true">Event - ‘Before Shakespeare’ at The National Archives</h3>
-                <p><time datetime="2018-08-01">Wednesday 1 August 2018</time></p>
+                <h3 class="card-group-secondary-nav__heading" aria-hidden="true">Event - ‘Before Shakespeare’ at The National Archives</h3>
```

Comment:
Just a quick question - why are we hiding this from assistive technology with `aria-hidden`?

Reply:
Hi @JamesWChan, would you be able to answer this?

Reply:
I copied this code directly from card-group-secondary-nav.php, I haven't changed it. There is an`<h3>` on line 4 with a class of `sr-only` so I guess this is to prevent screen readers from reading the heading out twice. If you want it changed just let me know.

Reply:
Thanks @JamesWChan 

I'll also need to update this file so that the card is an `<li>` instead of a `<div>`, as we made all the cards sit within a `<ul>` in `ds-wagtail` but I didn't reflect this in `ds-wagtail-frontend` (apologies) - it shouldn't affect anything visually on insights as the SASS in this repo is accounting  for the `<ul>` version of the cards.

Reply:
No worries 👍

Reply:
I've been thinking about the doubling of `<h3>` elements within this component and have spoken to @matt-blair about it. Matt has said that he sees the images as being decorative, in which case we can treat them as such - see https://www.w3.org/WAI/tutorials/images/decorative/ 

If we do that we would not need the initial `<h3>` and could treat the second one as a normal heading. This seems OK to me.

@AshTNA @JamesWChan - what do you think?

Reply:
Thanks @gtvj - that makes sense to me, though I don't know much about this component in general (aside from the card stuff)

Reply:
Sounds good to me. In which case, should we also remove the alt text as per the W3 advice?

Reply:
Yes - we'd need to set `alt=""` and remove the heading. If we're agreed, I think we should proceed with this change for all cards that have this duplicate heading pattern.